dm/dm/config/security.go

```diff
@@ -82,18 +82,33 @@ func (s *Security) LoadTLSContent() error {
 func (s *Security) DumpTLSContent(baseDirPath string) error {
 	if s.SSLCA == "" || !utils.IsFileExists(s.SSLCA) {
 		s.SSLCA = path.Join(baseDirPath, "ca.pem")
+		if !utils.IsDirExists(baseDirPath) {
+			if err := os.MkdirAll(baseDirPath, 0o700); err != nil {
+				return err
+			}
+		}
 		if err := utils.WriteFileAtomic(s.SSLCA, s.SSLCABytes, 0o600); err != nil {
 			return err
 		}
 	}
 	if s.SSLCert == "" || !utils.IsFileExists(s.SSLCert) {
 		s.SSLCert = path.Join(baseDirPath, "cert.pem")
+		if !utils.IsDirExists(baseDirPath) {
```

Comment:
ok, i will add a new function.

Reply:
when i fix this function, i notice `utils.IsDirExists` will return false when the path is a file, then it will return error when `os.MkdirAll`, and this error may be as expect.